_raml/apps/carts/carts.raml

```diff
@@ -202,3 +202,18 @@ post:
             application/json:
               example: !include examples/delete_carts_cartid_shipping_response.json
               schema: !include schema/delete_carts_cartid_shipping.schema.json
+
+  /shipping-data/shipping-method:
+    displayName: Shipping method
+    put:
+      description: Modifies the shipping method for a cart. *epages6 *epagesNow
```

Comment:
@HgSlv I've kept it short and simple for now (like for the other endpoints). If there are other specifications, please tell me (for example if the shipping method can only be updated under certain conditions/ for specific carts :)

Reply:
only shipping methods that can be used at checkout can be used for a cart.
This is consistent with the endpoint that returns all shipping-methods (returns only those that can be used for a cart)